src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -322,5 +322,16 @@ static void RemoveBackgroundLayer(this UIView control)
 				}
 			}
 		}
+
+		public static void ClearSubviews(this UIView view)
+		{
+			var toRemove = new UIView[view.Subviews.Length];
+			view.Subviews.CopyTo(toRemove, 0);
```

Comment:
I think `Subviews` is an array and already is a copy as a result of the property usage?

Reply:
Normally I don't think so, but maybe here because it's a projection of an NSArray? 
Anyway, I was just blind converting the Linq-ish version from ContentViewHandler, but I've simplified it in the latest commit.